apps/server/src/modules/authorization/rule-manager.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable, InternalServerErrorException, NotImplementedException } from '@nestjs/common';
+import { User } from '@shared/domain/entity/user.entity';
+import { CourseGroupRule } from './rules/course-group.rule';
+import { CourseRule } from './rules/course.rule';
+import { LessonRule } from './rules/lesson.rule';
+import { SchoolExternalToolRule } from './rules/school-external-tool.rule';
+import { SchoolRule } from './rules/school.rule';
+import { SubmissionRule } from './rules/submission.rule';
+import { TaskCardRule } from './rules/task-card.rule';
+import { TaskRule } from './rules/task.rule';
+import { TeamRule } from './rules/team.rule';
+import { UserRule } from './rules/user.rule';
```

Comment:
Why not "group" this imports all together into?
```js
import {
  CourseGroupRule,
  CourseRule,
  LessonRule,
  SchoolExternalToolRule,
  SchoolRule,
  SubmissionRule,
  TaskCardRule,
  TaskRule,
  TeamRule,
  UserRule,
} from './rules';
```

Reply:
Just overlooked that and VSCode's "Organize Imports" command, which I normally use, does not resolve it. If you find other places, please comment.